src/Config/Cleanup.targets

```diff
@@ -8,6 +8,16 @@
       <FilesToDelete Include="$(OutputPath)nodes\SharpDX*"/>
       <FilesToDelete Include="$(OutputPath)nodes\HelixToolkit*"/>
       <FilesToDelete Include="$(OutputPath)nodes\Cyotek*"/>
+      <FilesToDelete Include="$(OutputPath)nodes\Newtonsoft.Json*"/>
```

Comment:
do all of these binaries also end up in the bin folder @alfredo-pozo ?

Reply:
All those binaries are deleted from bin\Release\nodes folder and they already exist on bin\Release, I've already compared that to the current build, that's how I find that those nuget package binaries didn't exist on that subfolder (nodes)